core/src/main/java/org/elasticsearch/action/admin/indices/template/put/PutIndexTemplateRequest.java

```diff
@@ -473,7 +471,7 @@ public void readFrom(StreamInput in) throws IOException {
         cause = in.readString();
         name = in.readString();
 
-        if (in.getVersion().onOrAfter(V_5_1_0)) {
+        if (in.getVersion().onOrAfter(Version.V_6_0_0_alpha1)) {
```

Comment:
BWC tests caught this like on the spot - the cluster lost the master all the time :)